api/iode.h

```diff
@@ -807,7 +804,7 @@ typedef char    ONAME[K_MAX_NAME + 1];
 typedef long    OSIZE;   /* JMP 23-05-00 */
 
 typedef struct  _kobj_ {
-    SWHDL       o_val;          // "position" in the memory -> to be passed to SW_getptr()
+    SWHDL       o_val;          // Handle of the object in the scr4/swap memory -> to be passed to SW_getptr()
```

Comment:
Handle -> Handle ("position") ?
I'm not sure how to translate handle in French ;)

Reply:
The term "handle" in the windows vocabulary identifies a windows object like a dialog, a menu, an allocated memory, an open file...